packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerScenario.ts

```diff
@@ -177,31 +222,49 @@ export function runUnitTestScenario(
 		const joiners: TestEditManager[] = [];
 		/**
 		 * Local helper to update all the state that is dependent on the sequencing of new edits.
+		 * Note that all commits are part of the same bunch and have the same session ID, sequence number
```

Comment:
This seems like a requirement rather than a general truth.
```suggestion
		 * Note that all commits must be part of the same bunch and have the same session ID, sequence number
```

Reply:
Yes, I thought that's what the comment meant :-).